test/dialect/oracle/test_compiler.py

```diff
@@ -1659,6 +1659,26 @@ def test_double_to_oracle_double(self):
             cast(column("foo"), d1), "CAST(foo AS DOUBLE PRECISION)"
         )
 
+    @testing.combinations(
```

Comment:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5550